lib/api/index.js

```diff
@@ -360,13 +361,19 @@ app.post('/:installationId/migrate/:undo?', [
   try {
     if (req.params.undo) {
       await jiraClient.devinfo.migration.undo();
+
+      const responseMsg = isProd() ? 'Successfully called migrationUndo' : 'Sync failed';
```

Comment:
'Sync Failed' isn't right though just because you're in non-prod. I think it would be good to just leave it as `Successfully called migrationUndo` for all calls, no matter if in prod or not.

Any errors will be caught by the try/catch, which now thinking about it, we probably shouldn't wrap it around both undo or complete call, but just have a try/catch for each one so we can handle the failure event differently.

Reply:
How would you recommend testing? We now call `jiraClient.devinfo.migration.undo()` and `jiraClient.devinfo.migration.complete()` for all environments and only call `instance.post` if in prod. From Splunk, I was able to confirm this request was only made from prod, but wasn't able to write a test for it (always failed saying it wasn't called).

Reply:
you can set `process.env.NODE_ENV` in the `beforeEach` and set it back to the original in `afterEach`.

Reply:
Already doing that.

Reply:
and no bueno?